libdrgn/python/thread.c

```diff
@@ -0,0 +1,122 @@
+#include "drgnpy.h"
+#include "helpers.h"
+
+static void Thread_dealloc(Thread *self)
+{
+	Py_DECREF(self->prog);
+	Py_TYPE(self)->tp_free((PyObject *)self);
+}
+
+static PyLongObject *Thread_get_tid(Thread *self)
+{
+	return (PyLongObject *)PyLong_FromUnsignedLong(self->thread->tid);
```

Comment:
No need for the cast if you just make this return a `PyObject *`:
```suggestion
static PyObject *Thread_get_tid(Thread *self)
{
	return PyLong_FromUnsignedLong(self->thread->tid);
```